Maker-Agents-Suit-Module/OSMAgent/package.json

```diff
@@ -15,7 +15,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "@nethermindeth/general-agents-module": "^1.0.5",
+    "@nethermindeth/general-agents-module": "github:NethermindEth/general-agents-module",
```

Comment:
Didn't the previous one work for you?

Reply:
nope, it throws error and says didn't find module